kibana/readonlyrest-api.md

```diff
@@ -10,10 +10,4 @@ As an Enterprise user, you can benefit from automating the security configuratio
 
 Every request is **validated against ROR syntax first**, and rejected if syntactically or semantically incorrect. This adds to the safety of each change operation.
 
-{% swagger src="https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml" path="/api/ror/settings" method="post" %}
-[https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml](https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml)
-{% endswagger %}
-
-{% swagger src="https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml" path="/api/ror/settings/upload" method="post" %}
-[https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml](https://raw.githubusercontent.com/beshu-tech/readonlyrest-docs/develop/.gitbook/assets/ReadonlyREST-api.yml)
-{% endswagger %}
+[Link to the API documentation](https://api.beshu.tech/docs/swagger/master)
```

Comment:
@sscarduzio It would be great to have this URL conditionally based on the domain/environment, but it's probably not possible in the GitBook.

Reply:
I could try to check on the origin header (if any) 🤔